kcli_pre.sh

```diff
@@ -134,3 +134,18 @@ echo argocd_clusters_app_path needs to be set for argocd && exit 1
 echo argocd_policies_app_path needs to be set for argocd && exit 1
 {% endif %}
 {% endif %}
+
+## Cleaning
+CLUSTER={{ cluster }}
+POOL={{ pool }}
+POOLPATH=$(kcli list pool | grep $POOL | cut -d"|" -f 3 | xargs)
+export LC_ALL="en_US.UTF-8"
+find $POOLPATH/boot-* -type f -mtime +5 -exec sh -c 'virsh vol-delete {} || rm {}' \;
+VMS=$(kcli list vm -o json | jq ".[] | select(.name|test("$CLUSTER")) | .name" | xargs)
```

Comment:
I think need to install `jq` for that on all machines.

Reply:
1. `kcli` has a problem when prints to output JSON or YAML, it prints line `ESC[33mUsing local as no client was specified...ESC[0m` to stdout, instead of stderr, this breaks parsing.
```bash
$ kcli list vm -o json | jq ".[] | select(.name|test(\"cnfdc2\")) | .name"
parse error: Invalid numeric literal at line 1, column 2
```
2. Need to escape quotes: `test(\"$CLUSTER\")`

Result:
```bash
$ kcli list vm -o json | sed -n "2,$"p | jq ".[] | select(.name|test(\"cnfdc2\")) | .name"
"cnfdc2-installer"
"cnfdc2-master-0"
"cnfdc2-master-1"
"cnfdc2-master-2"


```

Reply:
1 addressed in https://github.com/karmab/kcli/commit/c988c3837bfa97415dfd8511eadf5666c365aef4 although well keep your workaround for retrocompatiblity purposes
2 forgot to escape indeed